dev_mode/index.js

```diff
@@ -212,7 +212,9 @@ export async function main() {
   var devMode = (PageConfig.getOption('devMode') || '').toLowerCase() === 'true';
 
   if (exposeAppInBrowser || devMode) {
+    // This is deprecated in favor of more generic window.jupyterapp
     window.jupyterlab = lab;
+    window.jupyterapp = lab;
```

Comment:
For completeness, maybe we can mention the new `window.jupyterapp` in the `expose_app_in_browser` help:

https://github.com/jupyterlab/jupyterlab/blob/3c04b9f5cfa8b9a04ce71974493f13eba4bf2c9e/jupyterlab/labapp.py#L541

Reply:
Thanks the help message has been updated.